tests/integration/components/content-test.js

```diff
@@ -0,0 +1,68 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import { assertTooltipContent } from '../../helpers/ember-tooltips';
+
+moduleForComponent('tooltip-on-element', 'Integration | Option | content', {
+  integration: true,
+});
+
+test('assertTooltipContent correctly matches expected tootltip content', function(assert) {
+
+  assert.expect(2);
+
+  this.render(hbs`
```

Comment:
To stay consistent with our other tests I think we should remove the `<div>` and use `this.render(hbs`{{tooltip-on-element test="foo"}}`);`.